apps/src/music/blockly/MusicBlocklyWorkspace.js

```diff
@@ -84,7 +84,7 @@ export default class MusicBlocklyWorkspace {
     Blockly.fieldRegistry.register(FIELD_CHORD_TYPE, FieldChord);
 
     this.workspace = Blockly.inject(container, {
-      toolbox: getToolbox(toolboxAllowList),
+      toolbox: getToolbox(),
```

Comment:
Does the toolbox still use the allow list on the first displayed level?

Reply:
Yep - we actually weren't passing in this parameter at all anymore, but instead setting the toolbox via updateToolbox when level data first loads.